fdbclient/GlobalConfig.actor.cpp

```diff
@@ -231,7 +247,8 @@ ACTOR Future<Void> GlobalConfig::refresh(GlobalConfig* self) {
 			TraceEvent("GlobalConfigRefreshError").errorUnsuppressed(e).suppressFor(1.0);
 			wait(tr->onError(e));
 			tr.clear();
-			wait(delay(0));
+			// tr is cleared, so it won't backoff properly. Use custom backoff logic here.
+			wait(backoff.onError());
```

Comment:
This transaction is not the only one that uses this pattern (see native API, for example), so perhaps this mechanism should be generalized and used elsewhere that we do this transaction refreshing.